addons/mail/models/models.py

```diff
@@ -198,6 +198,12 @@ def _get_default_activity_view(self):
     # GATEWAY: NOTIFICATION
     # ------------------------------------------------------------
 
+    @api.model
```

Comment:
Does not seem to be a model (notably in helpdesk, you check self -> this is a standard multi whose recordset could be void).